GMX-Bots/Detect-many-closing/README.md

```diff
@@ -0,0 +1,33 @@
+# Detects Many Position Closing From An Account Within A Time-Frame.
+
+## Description
+
+Returns a finding every time an account closes many positions in a certain amount of time.
+
+## Supported Chains
+
+- Arbitrum
+- Avalanche
+
+## Alerts
+
+- GMX-4
+  - Fired when `DecreasePosition` event is emitted on `Vault` contract multiple time in a time-frame.
+  - Severity is always set to "Medium".
+  - Type is always set to "Suspicious".
+  - Metadata contains:
+    - `account`: User address account
+    - `numberOfClosing`: The number of closed positions
+
+## Test Data
+
+Kovan Testnet (PoC)
+The following test was generated by interacting with PoC contracts (available at the PoC/ directory) deployed on the Kovan testnet.
+
+To check the results, uncomment the lines indicated in src/agent.config.ts, set a Kovan testnet RPC (e.g. https://kovan.poa.network) as jsonRpcUrl in your forta.config.json file and run:
+
+```
+npm run block 32228807
+```
+
+In the block 32228807, `Vault` contract generated 50 closing in only one transaction
```

Comment:
Block not generating findings for me

Reply:
Ok I will chek👍

Reply:
@yasmine9 so sorry for that the address on the `network.ts` was incorrect 😅 it was same as Avaax network. Its working now